.github/workflows/build.yml

```diff
@@ -126,3 +126,13 @@ jobs:
           tags: |
             ${{ env.IMAGE_PREFIX }}:${{ env.IMAGE_TAG }}
             ${{ env.IMAGE_PREFIX }}:latest
+
+      - name: Build binaries
+        run: make build-orb-cli-binaries
+
+      - uses: actions/upload-artifact@v2
```

Comment:
GitHub Artifacts are transient. What are these being used for?

We are better off tagging.

Reply:
just need it until we tagging